dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -629,4 +635,14 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         return S_OK;
     }
     CATCH_RETURN();
+
+    IFACEMETHODIMP PushNotificationManager::OnToastNotificationReceived(
+        ToastNotifications::INotificationProperties* notificationProperties,
+        ToastNotifications::INotificationTransientProperties* notificationTransientProperties) noexcept try
+    {
+        DWORD notificationId{ 0 };
+        ToastNotifications_PostToast(PushNotificationHelpers::GetAppUserModelId().get(), notificationProperties, notificationTransientProperties, &notificationId);
```

Comment:
Is this call only applicable for Packaged Apps? It seems to be the case. If so, I would recommend having a check in the beginning of the function:
THROW_HR_IF(E_UNEXPECTED, !IsPackaged());

Reply:
Actually even better:
THROW_HR_IF(E_UNEXPECTED, !IsPackagedAppScenario());